CHANGELOG.next.asciidoc

```diff
@@ -93,6 +93,8 @@ TLS or Beats that accept connections over TLS and validate client certificates.
 - Add DynamoDB AWS Metricbeat light module {pull}15097[15097]
 - Release elb module as GA. {pull}15485[15485]
 - Add a `system/network_summary` metricset {pull}15196[15196]
+- Add mesh metricset for Istio Metricbeat module {pull}15535[15535]
```

Comment:
Is this line expected here?

Reply:
yeap, the link was broken previously and i noticed that here 😬

Reply:
Oh, I see :+1: